### PR TITLE
[VnV Plan] Add system tests for employee management

### LINKED ISSUE #181: [VnV Plan] Functional Requirements System Tests - Employee Management
**Report:** Verification and Validation Plan
**Section Number:** 5.1

### Rubric Sections
- System Tests for Functional Requirements are specific
- Nondynamic testing  used as necessary
- Quality System Tests for Funct and NonFunc Requirements
- LO_Plan: Selects appropriately from relevant knowledge base to plan appropriate data collection methods and analysis strategies

### PR DESCRIPTION
**Pull request by LeonSo7**
[VnV Plan] Add system tests for employee management

Close #181

### CODE REVIEW

File: docs/VnVPlan/VnVPlan.tex [hunk 1]
```diff
@@ -782,6 +783,107 @@ \subsubsection{Work Orders}
...
+This section will contains tests covering the ``Employee Management'' requirements defined in the
+\href{https://github.com/arkinmodi/project-sayyara/blob/main/docs/SRS/SRS.pdf}{SRS}.
+
+\begin{enumerate}
+
+	\item \textbf{FRT-BE24-1}
+
+	      Control: Manual
+
+	      Initial State: Shop owner with no employees invited to shop yet
```
Comment (arkinmodi) [inline docs/VnVPlan/VnVPlan.tex:797]:
```suggestion
	      Initial State: Shop owner account with no employees invited to shop yet
```

File: docs/VnVPlan/VnVPlan.tex [hunk 2]
```diff
@@ -782,6 +783,107 @@ \subsubsection{Work Orders}
...
+	      system shall send an invitation email to the invited employee(s)
+
+	      How test will be performed: The tester will use a shop owner account to send employee invitation
+	      emails
+
+	\item \textbf{FRT-BE24-2}
+
+	      Control: Manual
+
+	      Initial State: Employee with invitation email
```
Comment (arkinmodi) [inline docs/VnVPlan/VnVPlan.tex:813]:
```suggestion
	      Initial State: Employee account with invitation email
```

File: docs/VnVPlan/VnVPlan.tex [hunk 3]
```diff
@@ -782,6 +783,107 @@ \subsubsection{Work Orders}
...
+
+	      Test Case Derivation: The system shall allow the user to accept an invitation
+
+	      How test will be performed: The tester will attempt to accept an employee invitation email
+
+	\item \textbf{FRT-BE25-1}
+
+	      Control: Manual
+
+	      Initial State: Shop owner with employees listed under the shop
```
Comment (arkinmodi) [inline docs/VnVPlan/VnVPlan.tex:827]:
```suggestion
	      Initial State: Shop owner account with employees listed under the shop
```

File: docs/VnVPlan/VnVPlan.tex [hunk 4]
```diff
@@ -782,6 +783,107 @@ \subsubsection{Work Orders}
 
 \end{enumerate}
 
+\subsubsection{Employee Management}
+
+This section will contains tests covering the ``Employee Management'' requirements defined in the
```
Comment (TimChoy) [inline docs/VnVPlan/VnVPlan.tex:788]:
```suggestion
This section will contain tests covering the ``Employee Management'' requirements defined in the
```

Comment (TimChoy) [reply docs/VnVPlan/VnVPlan.tex:788]: I fixed the ones that have been push to main already

**Review by arkinmodi — APPROVED** [review]

**Pull request merged**